clusterloader2/testing/experimental/storage/pod-startup/persistent-volumes/config.yaml

```diff
@@ -0,0 +1,142 @@
+# ASSUMPTIONS:
+# TODO: Assumptions
+
+# Constants
+  {{$NODE_MODE := DefaultParam .NODE_MODE "allnodes"}}
+  {{$DEFAULT_NODES_PER_NAMESPACE := DefaultParam .NODES_PER_NAMESPACE 100}}
+
+# Variables
+  {{$PODS_PER_NODE := DefaultParam .PODS_PER_NODE 50}}
+  {{$PVC_PER_POD := DefaultParam .PVC_PER_POD 2}}
+  {{$ENABLE_CHAOSMONKEY := DefaultParam .ENABLE_CHAOSMONKEY false}}
+  {{$TEST_NAME := DefaultParam .TEST_NAME "generic-test"}}
+  {{$PVC_SIZE := DefaultParam .PVC_SIZE "8Gi"}}
```

Comment:
Why did we pick `8Gi`

Reply:
Its a nice factor of 2 :)

In honestly there is not a reason behind it other than it seems like a reasonable choice for a single mid sized PV to be used as the default.